lxd/network/openvswitch/ovn.go

```diff
@@ -178,34 +178,68 @@ func NewOVN(s *state.State) (*OVN, error) {
 		return nil, errors.Wrapf(err, "Failed to get OVN northbound connection string")
 	}
 
+	sbConnection, err := NewOVS().OVNSouthboundDBRemoteAddress()
```

Comment:
You indeed never check `err` here.

Reply:
Prior to this branch, the function would return `return client, err` which was taking care of the check that way.

I actually much prefer the `return client, nil` but we do need to add a:
```
	if err != nil {
		return nil, errors.Wrapf(err, "Failed to get OVN southbound connection string")
	}
```

So that we properly handled errors.

Reply:
Ok sorted now.